gcp_variant_transforms/testing/integration/run_tests.py

```diff
@@ -188,9 +189,15 @@ def run_assertion(self):
 class QueryFormatter(object):
   """Formats a query.
 
-  Replaces keyword TABLE_NAME and eventually macros in the query.
+  Replaces macros and variable TABLE_NAME in the query.
```

Comment:
nit: by the same token as Bashir's comment, please remove 'table_name' from here as well. (just "Replaces macros and variables in the query").

Reply:
Done.